Sorting_visualization/Sort_vi.py

```diff
@@ -0,0 +1,595 @@
+import pygame
```

Comment:
Please add this file in the right folder:
`Python/Sorting_visualization/Sort_vi.py`